.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+rvm:
+- 2.5.1
+before_install:
+- openssl aes-256-cbc -K $encrypted_4d6950ff1422_key -iv $encrypted_4d6950ff1422_iv
```

Comment:
Does this need to be guarded by 
```
if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]
```
as well?

Reply:
(Since the variable will be unset when testing PRs.)

Reply:
This is the recommended Travis way to set things up. I could put the decrypt in the guard clause if you want, in which case it wouldn't decrypt the ssl key unless it really needed to.

What you're seeing here is the line as travis itself adds it.